### DISCUSSION TITLE
Passing `muted` as `false` pauses `BackgroundVideo`

### DISCUSSION THREAD

**Discussion by lemomar** [muxinc/next-video#320] [Q&A]
Hi there! I'm facing an issue with `BackgroundVideo` where setting `muted` to `false` pauses the video. Is there any way around this?

Code sample to reproduce:
```
export const BackgroundVideoWithPauseIssue = ({url}:{url:string}) => {
  return <BackgroundVideo src={url} muted={false} />;
}
```

**Comment by luwes** [comment]
you could try `<BackgroundVideo src={url} muted={false} autoplay={'any'} />`

but it's still up to the browser to allow autoplay with sound. it depends from user to user.
see https://developer.chrome.com/blog/autoplay
